README.md

```diff
@@ -232,3 +232,78 @@ package with features:
       -- code of the function
   end)
   ```
+
+## Tarantool 3.0 role
+
+`roles.expirationd` is a Tarantool 3.0 role for the expirationd
+package with features:
+
+* You can configure the expirationd role with `cfg` entry (check example).
+  cluster configuration allows to set the same parameters as
+  in [expirationd.cfg()](https://tarantool.github.io/expirationd/#cfg)
+* You can use persistent functions (i.e. created by `box.schema.func.create`).
+  When configuring, role tries firstly get function from global namespace
+  (`_G`) and if function was not found then role tries search in `box.func` for
+  function with the same name.
+  If some functions from config are missing, expirationd will wait for their
+  creation and start tasks when all of them are found. You can check alerts
+  to see what functions are missing.
+* The role stops all expirationd tasks on an instance on the role termination.
+* The role can automatically start or kill old tasks from the role
+  configuration.
+
+  ```yaml
+  roles: [roles.expirationd]
+    roles_cfg:
+      roles.expirationd:
+        cfg:
+          metrics: true
+        task_name1:
+          space: users
+          is_expired: is_expired_func_name
+          is_master_only: true
+          options:
+            args:
+            - any
+            atomic_iteration: false
+            force: false
+            force_allow_functional_index: true
+            full_scan_delay: 1
+            full_scan_time: 1
+            index: 0
+            iterate_with: iterate_with_func_name_in__G
+            iteration_delay: 1
+            iterator_type: ALL
+            on_full_scan_complete: on_full_scan_complete_func_name_in__G
+            on_full_scan_error: on_full_scan_error_func_name_in__G
+            on_full_scan_start: on_full_scan_start_func_name_in__G
+            on_full_scan_success: on_full_scan_success_func_name_in__G
+            process_expired_tuple: process_expired_tuple_func_name_in__G
+            process_while: process_while_func_name_in__G
+            start_key:
+            - 1
+            tuples_per_iteration: 100
+            vinyl_assumed_space_len: 100
+            vinyl_assumed_space_len_factor: 1
+  ```
+
+  [expirationd.start()](https://tarantool.github.io/expirationd/#start) has
+  the same parameters with the same meaning except for the additional optional
+  param `is_master_only`. If `true`, the task should run only on a master
+  instance. By default, the value is `false`.
+
+  You need to be careful with parameters-functions. Task will not start until it
```

Comment:
Maybe "function parameters" would be a less bit confusing (feel free to ignore if you disagree.)

Reply:
changed